packages/amazonq/.changes/next-release/Feature-24aa4962-f8c1-4371-9725-216477e036dd.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Pair Programmer Mode: Amazon Q can now write code and run shell commands on your behalf"
```

Comment:
Should we quickly review this with Ankit to see if he has any opinions? Other variations could be:
```
Agentic chat: Amazon Q can now write code and run shell commands on your behalf
```